cmd/deploy/deploy_test.go

```diff
@@ -1165,3 +1167,44 @@ func TestTrackDeploy(t *testing.T) {
 		})
 	}
 }
+
+func TestOktetoManifestPathFlag(t *testing.T) {
+	opts := &Options{}
+	var tests = []struct {
+		name        string
+		manifest    string
+		expectedErr error
+	}{
+		{
+			name:        "manifest file path exists",
+			manifest:    "okteto.yml",
+			expectedErr: nil,
+		},
+		{
+			name:        "manifest file path doesn't exist",
+			manifest:    "nonexistent.yml",
+			expectedErr: fmt.Errorf("nonexistent.yml file doesn't exist"),
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			dir, err := os.Getwd()
+			assert.NoError(t, err)
+			fullpath := filepath.Join(dir, tt.manifest)
+			opts.ManifestPath = fullpath
+			if tt.manifest != "nonexistent.yml" {
+				// create the manifest file only if it's not the nonexistent scenario
+				f, err := os.Create(fullpath)
```

Comment:
This will create an actual file in the host running the tests.

It's best practice that all automated test have no side effects on the host. In this case the test file is not cleaned so it can be considered "garbage" for the host.

In other parts of the CLI we use `afero` to create virtual filesystems which makes testing much simpler because you don't have to worry about this cleanup for example. That would be the ideal approach.

If that cannot be implemented for any reason, I would make sure to use a tmp directory and cleanup on completion.

Reply:
Implemented! I maybe wrong in using `afero` package here (sorry, if i'm) but let me know if there are changes required.